app/src/test/java/com/mapzen/erasermap/view/MainActivityTest.kt

```diff
@@ -0,0 +1,494 @@
+package com.mapzen.erasermap.view
+
+import android.app.AlertDialog
+import com.mapzen.android.lost.api.LocationServices
+import com.mapzen.erasermap.BuildConfig
+import com.mapzen.erasermap.PrivateMapsTestRunner
+import com.mapzen.erasermap.R
+import com.mapzen.pelias.SavedSearch
+import com.mapzen.pelias.gson.Feature
+import com.mapzen.pelias.widget.PeliasSearchView
+import com.mapzen.tangram.MapView
+import com.mapzen.valhalla.Route
+import com.mapzen.valhalla.Router
+
+import org.json.JSONObject
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.Robolectric
+import org.robolectric.annotation.Config
+import org.robolectric.fakes.RoboMenu
+import org.robolectric.shadows.ShadowActivity
+import org.robolectric.shadows.ShadowIntent
+import org.robolectric.shadows.ShadowLocationManager
+
+import android.content.Intent
+import android.content.SharedPreferences
+import android.location.LocationManager
+import android.os.SystemClock
+import android.preference.PreferenceManager
+import android.support.v7.widget.SearchView
+import android.view.Menu
+import android.view.MotionEvent
+import android.view.View
+import android.widget.TextView
+
+import java.util.ArrayList
+
+import android.content.Context.LOCATION_SERVICE
+import android.content.DialogInterface
+import android.view.View.GONE
+import android.view.View.VISIBLE
+import com.mapzen.erasermap.dummy.TestHelper.getFixture
+import com.mapzen.erasermap.dummy.TestHelper.getTestFeature
+import com.mapzen.erasermap.dummy.TestHelper.getTestLocation
+import com.mapzen.erasermap.model.ManifestDownLoader
+import com.squareup.okhttp.mockwebserver.MockResponse
+import com.squareup.okhttp.mockwebserver.MockWebServer
+import org.assertj.core.api.Assertions.assertThat
+import org.robolectric.Shadows.shadowOf
+import org.robolectric.shadows.ShadowAlertDialog
+import java.util.concurrent.TimeUnit
+
+RunWith(PrivateMapsTestRunner::class)
+Config(constants = BuildConfig::class,  sdk=intArrayOf(21))
+public class MainActivityTest {
+    private var activity: MainActivity? = null
+    private var locationManager: LocationManager? = null
+    private var shadowLocationManager: ShadowLocationManager? = null
+
+    @Before
+    throws(Exception::class)
```

Comment:
Were these added by the Java -> Kotin converter? I believe all instances of `throws(Exception::class)` can just be deleted. Kotlin does not have the concept of checked exceptions.

http://kotlinlang.org/docs/reference/exceptions.html#checked-exceptions